### ISSUE TITLE
Test the repo documentation

### ISSUE THREAD

**Issue by claudia-lat** [ami-iit/paper_latella_2023_irim_muscle-force-estimation#1]
In this issue, the documentation and code of this repo are going to be tested.

**Comment by claudia-lat** [comment]
CC @lrapetti

**Comment by lrapetti** [comment]
> CC @lrapetti

Hi @claudia-lat, I went trough the documentation and it seems fine to me. Unfortunately I no longer have a Matlab licence so I cannot test the code.

**Comment by GiulioRomualdi** [comment]
I tested the code as documented in the README. It works fine. As discussed with @claudia-lat, in trial 1 the subject doesn't move probably because the motions are taken from a calibration of the sensors. Trial 5 works as expected :)

**Comment by claudia-lat** [comment]
As @GiulioRomualdi noticed, if the user launches the trial 1 the subject seems not to move.  This because Trial 1 and 2 are actually acquisitions for the MVC (i.e., Maximum Voluntary Contraction) as described in https://huggingface.co/datasets/ami-iit/paper_latella_2023_irim_muscle-force-estimation_dataset/blob/main/dataset/README.md thus the subject in these two trials is not performing the plantarflexion/dorsiflection tasks but excerting the MVC for the muscles. 
This means that the actual motion trials are Trial 3, 4 and 5. 
I've added a note in f99e0efda472b7de768d2f2233a5c024a8e39500.

Thanks @GiulioRomualdi for testing the documentation.
Closing this issue.
